native/rust/archetype/src/main/resources/archetype-resources/src/main/rust/Cargo.toml

```diff
@@ -9,13 +9,20 @@
 # See the NOTICE file distributed with this work for information regarding copyright ownership.
 #
 [package]
-name = "${cargo_crate_name}"
-edition = "2021"
+name = "rspdk"
 version = "0.1.0"
+edition = "2021"
 license = "Apache-2.0"
 authors = [ "Alluxio Developers" ]
 
 [lib]
 crate-type = [ "${cargo_crate_type}" ]
 
 [dependencies]
+spdk-sys = { path = "spdk-sys" }
```

Comment:
seems like this have been archived and any reason stick to this dependency instead of [spdk-rs](https://github.com/openebs/spdk-rs)

Reply:
this is not the `spdk-sys` crate on crates.io, but a local crate generated from the C sources of SPDK.

Reply:
`src/main/rust/spdk-sys/Cargo.toml` it's this crate

Reply:
yeah I know, just seems like spdk-sys is archived and same team continue to work on https://github.com/openebs/spdk-rs